src/components/ErrorMessage.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
@paltman this is great, and I like what you're starting to introduce as far as CSS `var`-with-fallbacks.

I've added this to ![](https://github.trello.services/images/mini-trello-icon.png) [Review components to be extracted from Explore Homer](https://trello.com/c/J7Yc3HFk/137-review-components-to-be-extracted-from-explore-homer) to think about how we might reuse it (also see next comment below).